desktop/src/main/java/bisq/desktop/components/TitledGroupBg.java

```diff
@@ -77,8 +87,14 @@ public void setText(String text) {
         this.text.set(text);
     }
 
-    public Label getLabel() {
-        return label;
-    }
+    public void setupHelpUrl(String helpUrl) {
+        if (helpIcon == null) {
+            helpIcon = new Label();
+            AwesomeDude.setIcon(helpIcon, AwesomeIcon.QUESTION_SIGN, "1.2em");
+            helpIcon.getStyleClass().add("show-hand");
```

Comment:
As I wanted to get rid of the AwesomeIcon Font at some point of time I rather move it into the FormBuilder to make it easier for refactoring.